tests/e2e/collections/ansible_collections/e2e/tests/roles/rescue/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: "[ {{ test_identifier }} ] - Rescue/Debug"
```

Comment:
One small comment here, you should use `rescue_test_identifier` instead of `test_identifier`.